src/application/room/manage_room.service.ts

```diff
@@ -20,81 +21,82 @@ export class ManageRoomServiceImpl {
     private roomRepository: Repository<RoomModel>,
   ) {}
 
-  async createRoom(Room: CreateRoomRequestDTO, user) {
+  async createRoom(room: RoomDetailRequestDTO, user) {
     try {
-      const userId = await this.userService.findUserByEmail(user.email);
-      const room = new RoomModel();
-      room.name = Room.name;
-      room.capacity = Room.capacity;
-      room.price = Room.price;
-      room.status = Room.status;
-      room.description = Room.description;
-      room.image = Room.image;
-      room.rating = Room.rating;
-      room.address = Room.address;
-      room.user = userId;
-      room.location = `${Room?.location?.lat},${Room?.location?.lng}`;
-      return await this.roomRepository.save(room);
+      const currentUser = await this.userService.findUserByEmail(user.email);
+      const roomModel = RoomUtil.getRoomModel(room);
+      roomModel.user = currentUser;
```

Comment:
e đang thấy cái name lưu id của user trong bảng room là owner á a Vĩnh

Reply:
@TuanHai37 Chỗ ni mình truyền như thế này sẽ đảm bảo tính toàn vẹn dữ liệu.
Em có cách nào khác thì cứ suggest, nó tối ưu hơn thì mình làm.